include/cantera/thermo/Elements.h

```diff
@@ -84,6 +84,54 @@ namespace Cantera
 //! stable state at 298.15 K and 1 bar.
 #define ENTROPY298_UNKNOWN -123456789.
 
+//! Get the atomic symbols of the elements defined in Cantera.
+/*!
+ * @return A vector of the atomic symbols
+ */
+const std::vector<std::string>& elementSymbols();
+
+//! Get the names of the elements defined in Cantera.
+/*!
+ * @return A vector of the element names
+ */
+const std::vector<std::string>& elementNames();
+
+//! Get a map with the element symbols as keys and weights as values.
+/*!
+ * This is a constant in the application so it is only generated once
+ * when it is first needed.
+ *
+ * @return A map of string symbols to atomic weights
```

Comment:
I think most of these `@return` annotations are unnecessary -- they just repeat what is stated in the summary docstring. But if you want this, I'd suggest:
```suggestion
 * @return A map of element symbols to atomic weights
```